sunpy/util/tests/test_scraper.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+import datetime
+import os
+
+import sunpy.data.test
+from sunpy.time import TimeRange
+from sunpy.util.scraper import Scraper
+
+PATTERN_EXAMPLES = [
+    ('%b%y', datetime.timedelta(days=31)),
+    ('%m%y', datetime.timedelta(days=31)),
+    ('%H%d', datetime.timedelta(hours=1)),
+    ]
+
+class TestScraper:
```

Comment:
do you need the class? You don't seem to be sharing any variables etc? Just functions is probably better?